src/sacn/source.c

```diff
@@ -366,18 +922,61 @@ etcpal_error_t sacn_source_change_synchronization_universe(sacn_source_t handle,
 etcpal_error_t sacn_source_send_now(sacn_source_t handle, uint16_t universe, uint8_t start_code, const uint8_t* buffer,
                                     size_t buflen)
 {
-  ETCPAL_UNUSED_ARG(handle);
-  ETCPAL_UNUSED_ARG(universe);
-  ETCPAL_UNUSED_ARG(start_code);
-  ETCPAL_UNUSED_ARG(buffer);
-  ETCPAL_UNUSED_ARG(buflen);
-  return kEtcPalErrNotImpl;
+#if !SACN_SOURCE_ENABLED
+  return kEtcPalErrNotInit;
+#endif
+
+  etcpal_error_t result = kEtcPalErrOk;
+
+  // Verify module initialized.
+  if (!sacn_initialized())
+    result = kEtcPalErrNotInit;
+
+  // Check for invalid arguments.
+  if (result == kEtcPalErrOk)
+  {
+    if (handle == SACN_SOURCE_INVALID)
```

Comment:
Combine these checks into a single if() block

Reply:
Resolved in https://github.com/ETCLabs/sACN/pull/32/commits/4338ffdded0288b92cc19437c16825145d3966f0.